### ISSUE TITLE
 The background color for the statement doesn’t match the app

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#33978]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:**
**Reproducible in staging?:** needs reproduction (don't have statement access)
**Reproducible in production?:** needs reproduction (don't have statement access)
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @puneetlath 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1704219570242489

## Action Performed:
1. Click on the statement link from concierge chat
2. Observe the background color

## Expected Result:
The background color should match the app 

## Actual Result:
the header and body have different background colors

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

![image (9)](https://github.com/Expensify/App/assets/38435837/1b08f83d-4c19-4986-830b-42f1129079f8)

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0148c084d01887606e</li>
        <li>Upwork Job ID: 1744409181225820160</li>
        <li>Last Price Increase: 2024-01-08</li>
    </ul>
</details>

**Comment by MitchExpensify** [comment]
Reproduced!

**Comment by MitchExpensify** [comment]
Asking whether it should be internal or not here: https://expensify.slack.com/archives/C049HHMV9SM/p1704733013601179?thread_ts=1704219570.242489&cid=C049HHMV9SM

**Comment by rlinoz** [comment]
Hey @dubielzyk-expensify can you help us here?

We have this statement screen that was firstly designed to be in OldDot I think and we have a problem with the background mismatching the NewDot background color.

I was thinking that maybe we could just apply the new background colors to the old screen, but the dark mode gets pretty bad as per the screenshot attached, so I think we have a few options:

1 - Change the background to the light mode background and wait for a better day to maybe migrate this entire functionality to NewDot - This is the way it has been, but I think the new light mode made it more evident that the background shouldn't be the one it is.  This is low effort, but it doesn't look really good in dark mode.

2 - Update the theme in this page specifically to match the dark/light mode, but since this page is in OldDot it would be a whole new implementation of themes. - Medium effort, and maybe we will throw this out soon if we have plans for moving the statement screen to NewDot.

3 - Making this a new issue to move statements to NewDot and all use all our new components (maybe there is a new figma layout for this already?) - This is really high effort and we would probably have to take it to #whatsnext or check if it already planned for some wave.

I would appreciate your thoughts on this.

Dark mode:
<img width="386" alt="Screenshot 2024-01-10 at 12 39 53" src="https://github.com/Expensify/App/assets/5201282/acc874b3-3a01-4fd9-a1ce-3419761c7d41">

Light mode:
<img width="386" alt="Screenshot 2024-01-10 at 12 39 30" src="https://github.com/Expensify/App/assets/5201282/4dc4644c-5f63-4984-903c-dc21be5d9e63">

Dark mode with ligth background:
<img width="386" alt="Screenshot 2024-01-10 at 12 39 05" src="https://github.com/Expensify/App/assets/5201282/cd06da81-ce06-4f8f-a82e-82db090fc817">

**Comment by puneetlath** [comment]
Interesting considerations @rlinoz! It could be good to raise in #vip-split as that's where I think this is most relevant.

**Comment by dubielzyk-expensify** [comment]
Thanks for a great write up and reaching out @rlinoz. It's hard for me to comment as I'm unsure how severe and common this screen is and how much investment we'd wanna put into it. 

My gut reaction is that we should at least go with option 2 as option 1 breaks a lot of our UI guidelines and also the fact that the user has potentially selected dark mode.

While 3 would be the best option, I definitely think 2 feels like the right balance. Keen on @Expensify/design opinion on this as well

**Comment by shawnborton** [comment]
So I think we really just need to do a better job of making sure the statement view is using all of our standard CSS vars. Because OldDot is already fully ready for dark mode. Just go to www.expensify.com and type in `$('body').toggleClass('dark');` in the console

So that being said, once everything in the statement view uses the proper CSS vars, then I think we will be in a much better place. From here, I think it's really just a matter of being able to pass in some kind of url param into the statement url like `?theme=dark` or something which would add the body class of `dark` when needed.

**Comment by rlinoz** [comment]
I had no idea OldDot was ready for dark mode, that makes things way easier.

I guess statements is already using the correct CSS vars for colors, except for background which I had to change.

Does this looks correct? 
<img width="389" alt="image" src="https://github.com/Expensify/App/assets/5201282/829ce2ef-d068-411c-a82a-8d0bee0ba808">

<img width="395" alt="Screenshot 2024-01-11 at 13 00 59" src="https://github.com/Expensify/App/assets/5201282/d2a0d425-5466-4cf9-8dd1-b45924566e40">

**Comment by shawnborton** [comment]
That looks correct! While you are here though... something about the font doesn't quite look right. Notice how it feels squished? Any ideas?
![CleanShot 2024-01-11 at 19 44 23@2x](https://github.com/Expensify/App/assets/2319350/5c423309-1f6a-482f-931a-9f662cdd5ba1)

**Comment by shawnborton** [comment]
Also, the split bill icon is incorrect - it should be a receipt or arrows going left & right?

**Comment by rlinoz** [comment]
@shawnborton I couldn't find a left & right icon in our icons, but there is a receipt if that is ok.

About the font, we were using the `GT-America-Standard` I think we don't use that anymore? Anyway, I changed it to the one we are using in other pages.
<img width="393" alt="image" src="https://github.com/Expensify/App/assets/5201282/8a67f7cb-8bef-4c7c-a658-652ed081c1c5">

**Comment by shawnborton** [comment]
Nice! I think we're getting close. Can you double check that the font is correct though? For example, this part here:
![CleanShot 2024-01-11 at 20 13 44@2x](https://github.com/Expensify/App/assets/2319350/7c18546e-1031-4f38-b2a2-eb183ea91ff0)


If this was our correct font, we'd see a "double-story" g, like so:
![CleanShot 2024-01-11 at 20 14 28@2x](https://github.com/Expensify/App/assets/2319350/bbb56b44-598a-4fa4-92de-2894714cf4e9)

So I think this is the case for all of the type I see - it's not quite using our actual font.

**Comment by rlinoz** [comment]
Is this better? Should we change font sizes?
<img width="387" alt="image" src="https://github.com/Expensify/App/assets/5201282/cdfc4e27-d33f-4ae4-9394-a1fb34b59222">

**Comment by shawnborton** [comment]
That is definitely not correct, that looks like a default serif font.

**Comment by rlinoz** [comment]
Sorry about that, I think chrome was not playing nice and I have a hard time discerning fonts, but I think this one is correct 

Here is the font order being used `"Expensify-Neue", system-ui, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"


<img width="382" alt="Screenshot 2024-01-11 at 17 32 02" src="https://github.com/Expensify/App/assets/5201282/9a2b8748-0158-41ff-a56c-734d4686ccdf">
<img width="391" alt="Screenshot 2024-01-11 at 17 29 11" src="https://github.com/Expensify/App/assets/5201282/201e8e10-4f34-4749-adc6-7f8c630f314a">

**Comment by shawnborton** [comment]
Much better!

**Comment by rlinoz** [comment]
I forgot to add $ to the PR fixing this issue, and I am not sure how to trigger the payment automation for the C+ review.

Can you help us here @MitchExpensify, please?

**Comment by MitchExpensify** [comment]
**Payment summary:**

- C: N/A 
- C+: @sobitneupane $500 (NewDot)
- Reporter: N/A 

Please make the request in NewDot for compensation, @sobitneupane - Thank you!

**Comment by MitchExpensify** [comment]
Yes and this can be closed while we wait on that

**Comment by JmillsExpensify** [comment]
$500 approved for @sobitneupane based on [this summary](https://github.com/Expensify/App/issues/33978#issuecomment-1901033236).
